packages/orbit-components/.eslintrc.cjs

```diff
@@ -30,6 +30,10 @@ module.exports = {
       files: ["src/**/*.ts?(x)"],
       rules: {
         "@typescript-eslint/consistent-type-imports": "error",
+        "tailwindcss/no-custom-classname": [
+          1,
+          { whitelist: ["(orbit\\-).*"], config: "./tailwind.config.ts" },
```

Comment:
Why this change? I believe that if we want to use custom components in our docs, for example, where we have some, we will face same problem. So why can't we have it global?

Reply:
because it needs the reference to tailwind.config.ts, which is not global (it can't find our preset classes without it and eslint throws warnings) 
![Screenshot 2023-09-07 at 12 16 27](https://github.com/kiwicom/orbit/assets/14054752/3e516ad3-1a65-4c23-bb8b-073354eb8c01)

Reply:
I can try to have it globally and just import the object from our preset

Reply:
I don't think that is the reason. The warning is thrown every time a new class is used for the first time, until CSS is built. So if you see that warning what you can do is build the CSS `yarn components tailwind` and, if needed, restart the ESLint server on VSCode. Those steps have been enough to solve those warnings to me

Reply:
I thought it would eliminate the need to build the file

Reply:
yep, we can